cypress/package.json

```diff
@@ -3,6 +3,7 @@
   "name": "@klicker-uzh/cypress",
   "devDependencies": {
     "@cypress/code-coverage": "3.12.16",
+    "@klicker-uzh/prisma": "workspace:*",
```

Comment:
can we import the seed script from prisma to run it programmatically beforeEach? -> issues due to ESM